shell/platform/windows/window_win32_unittests.cc

```diff
@@ -231,5 +231,29 @@ TEST(MockWin32Window, KeyDownWithCtrlToggled) {
   SetKeyboardState(keyboard_state);
 }
 
+TEST(MockWin32Window, WindowOnMinimized) {
+  MockWin32Window window;
+  window.InjectWindowMessage(WM_SIZE, SIZE_RESTORED, MAKEWPARAM(1, 1));
+
+  EXPECT_CALL(window, OnMinimized()).Times(1);
+  window.InjectWindowMessage(WM_SIZE, SIZE_MINIMIZED, MAKEWPARAM(0, 0));
+
+  // notify only when window size is different.
+  EXPECT_CALL(window, OnMinimized()).Times(0);
+  window.InjectWindowMessage(WM_SIZE, SIZE_MINIMIZED, MAKEWPARAM(0, 0));
```

Comment:
Is this comment right? My understanding is that it only notifies if the type of resizing changes:

```suggestion
  // Notify only when type of resizing changes.
  EXPECT_CALL(window, OnMinimized()).Times(0);
  window.InjectWindowMessage(WM_SIZE, SIZE_MINIMIZED, MAKEWPARAM(0, 0));
```

Reply:
It looks nice! I applied this change. I'll mark this as resolved.